tests/installation/bootloader.pm

```diff
@@ -19,6 +19,12 @@ use registration;
 # hint: press shift-f10 trice for highest debug level
 sub run() {
     my ($self) = @_;
+    # arbitrary slow typing speed, also see bootloader_uefi
+    my $slow_typing_speed = 13;
+    # type even slower towards the end to ensure no keybuffer overflow even
+    # when scrolling within the boot command line to prevent character
+    # mangling
+    my $slower_typing_speed = 4;
```

Comment:
yes, "slower" should be even slower than "slow". I also saw these failures but don't know exactly how they are related. This commit should only affect everything below line 238. For all values in before this is just extracting the magic number